src/Support/Str.php

```diff
@@ -61,7 +61,8 @@ public static function evaluable(string $code): string
     {
         $code = self::PREFIX.str_replace(' ', '_', $code);
 
-        return (string) preg_replace('/[^A-Z_a-z0-9]/', '_', $code);
+        // sticks to PHP8.2 function naming rules https://www.php.net/manual/en/functions.user-defined.php
+        return (string) preg_replace('/[^a-zA-Z0-9_\x80-\xff]/', '_', $code);
```

Comment:
@nunomaduro I've changed this one to match PHP documentation rule for function names

https://www.php.net/manual/en/functions.user-defined.php